draft-ietf-tls-deprecate-obsolete-kex.md

```diff
@@ -245,7 +235,7 @@ Bleichenbacher's attack would affect all endpoints sharing the same RSA key (see
 
 Given these problems, this document updates {{!RFC9325}}, {{!RFC4346}}, {{!RFC5246}},
```

Comment:
I would delete "Given these problems, " since you add "remediate the above problems." at the end of the sentence.